src/containers/BookForm.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { createBook } from '../actions/index';
+
+class BookForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: null,
+      title: '',
+      category: '',
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit = event => {
+    const { createBook } = this.props;
+    event.preventDefault();
+    createBook(this.state);
+    event.target.reset();
+  }
+
+  handleChange = e => {
+    this.setState({
+      id: Math.floor(Math.random() * 50),
+      [e.target.name]: e.target.value,
+    });
+  }
+
+  render() {
+    const categories = [
+      'select',
+      'Action',
+      'Biography',
+      'History',
+      'Horror',
+      'Kids',
+      'Learning',
+      'Sci-Fi',
+    ];
+
+    const catList = categories.map(cat => (
+      <option key={Math.random()} value={cat}>
+        {cat}
+      </option>
+    ));
+
+    const { category } = this.state;
+
+    return (
+      <div>
+        <form onSubmit={this.handleSubmit}>
+          <label htmlFor="title">
+            Title
+            <input name="title" type="text" onChange={this.handleChange} />
```

Comment:
- [ ] Consider making the value of the input to read from state by setting `value = this.state.title`.

Reply:
Done.